edgedb-errors/src/display.rs

```diff
@@ -0,0 +1,97 @@
+use std::fmt;
+use std::str;
+use std::collections::BTreeMap;
+
+use bytes::Bytes;
+
+use crate::{Error, InternalServerError};
+
+
+pub const FIELD_HINT: u16 = 0x_00_01;
```

Comment:
These all should not be `pub` I think. Only two functions and two structs that they return.